### PR TITLE
MGMT-2314 Add cluster_id label to cluster related metrics

### PR DESCRIPTION
**Pull request by nmagnezi**

### CODE REVIEW

**Comment by nmagnezi** [comment]
/hold
running test-infra locally.

File: internal/metrics/metricsManager.go [hunk 1]
```diff
@@ -48,6 +48,7 @@ const (
 	namespace                  = ""
 	subsystem                  = "service"
 	openshiftVersionLabel      = "openshiftVersion"
+	clusterID                  = "clusterId"
```
Comment (gamli75) [inline internal/metrics/metricsManager.go:51]: please change the name to clusterIdLabel - it's a little bit confusing with the clusterId parameter.

**Review by gamli75 — COMMENTED** [review]
why you didn't add the cluster id to handleHostInstallationComplete method ? I think it's needed

File: internal/metrics/metricsManager.go (hunk 1 above)
Comment (nmagnezi) [reply internal/metrics/metricsManager.go:51]: ack

File: internal/host/host.go [hunk 2]
```diff
@@ -497,7 +497,7 @@ func (m *Manager) reportInstallationMetrics(ctx context.Context, h *models.Host,
 	if err != nil {
 		log.WithError(err).Errorf("not reporting installation metrics - failed to find cluster %s", h.ClusterID)
 	} else {
-		m.metricApi.ReportHostInstallationMetrics(log, cluster.OpenshiftVersion, h, previousProgress, CurrentStage)
+		m.metricApi.ReportHostInstallationMetrics(log, cluster.OpenshiftVersion, *cluster.ID, h, previousProgress, CurrentStage)
```
Comment (ronniel1) [inline internal/host/host.go:500]: @gamli75 , @romfreiman , @avishayt  if we have the clusterid for the host metrics, don't we also need hostId?

Comment (gamli75) [reply internal/host/host.go:500]: @ronniel1 I talked with @romfreiman about it and we agreed that we can add it later. The host HW is more important (VM ..)

**Comment by nmagnezi** [comment]
tested ok via test-infra, example: 

`service_assisted_installer_cluster_installation_seconds_bucket{clusterId="709b8035-1d20-4acb-bbd5-3c8bc9911347",openshiftVersion="4.6",result="installed",le="2100"} 1`

**Pull request merged**